src/bundlers/kotlin/kotlinMain.ts

```diff
@@ -0,0 +1,135 @@
+export const template = `
+/**
+* KOTLIN 
+* This is an auto generated code. This code should not be modified since the file can be overwriten
+* if new genezio commands are executed.
+*/
+package {{packageName}}
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.jsonObject
+import kotlinx.serialization.json.JsonElement
+import org.http4k.core.HttpHandler
+import org.http4k.core.Method.OPTIONS
+import org.http4k.core.Method.POST
+import org.http4k.core.Request
+import org.http4k.core.Response
+import org.http4k.core.Status.Companion.OK
+import org.http4k.core.then
+import org.http4k.filter.AllowAll
+import org.http4k.filter.CorsPolicy
+import org.http4k.filter.OriginPolicy
+import org.http4k.filter.ServerFilters
+import org.http4k.routing.bind
+import org.http4k.routing.routes
+import org.http4k.server.SunHttp
+import org.http4k.server.asServer
+import kotlinx.serialization.json.jsonArray
+import com.amazonaws.services.lambda.runtime.Context
+import com.amazonaws.services.lambda.runtime.RequestHandler
+
+@Serializable
+data class JsonRpcRequest(
+        val jsonrpc: String,
+        val id: Int,
+        val method: String,
+        val params: List<JsonElement>?,
+)
+
+@Serializable
+data class JsonRpcResponse(val jsonrpc: String, val id: Int, val result: String)
+
+class LambdaHandler : RequestHandler<Map<String, Any>, String> { 
+  override fun handleRequest(event: Map<String, Any>, context : Context): String {
+      println(event)
+      return processRequest(event)
+  }
+}
+
+fun processRequestDirect(json_rpc : String): String {
+  return processRequest(mapOf("body" to json_rpc))
+}
+
+fun processRequest(event: Map<String, Any>): String {
+    println("PROCESSING REQ")
+    // Get request body
+    var request: JsonRpcRequest = JsonRpcRequest("", 2, "", listOf())
+    var req_id = 0
+    try {
+        var eventBody = event["body"].toString().replace("\\\\", "")
+        if(eventBody.get(0) != '"') {
+          eventBody = "\\"" + eventBody + "\\""
+        }
+        request = Json.decodeFromString(eventBody.substring(1, eventBody.length - 1))
+        req_id = request.id
+    } catch (e: Exception) {
+        println(e)
+        return "{\\"jsonrpc\\":\\"2.0\\",\\"result\\":\\"There was a problem with the request format.\\",\\"id\\":\\"$req_id\\"}"
```

Comment:
This might not work on the latest `dev` branch.

After this fix was merged, `genezio local` expects a JSON object with a `body` field. You can take a look at the fix I did for dart [2]

[1] https://github.com/Genez-io/genezio/pull/322/files
[2] https://github.com/Genez-io/genezio/pull/329/files#diff-eba1dfefe0f2f26649b182df36f4a81fe459261d379f509dcd0403d2876d8fb3R160-R171